### ISSUE TITLE
Can't build proc-macro-hack 0.5.6 on Fedora Rawhide

### ISSUE THREAD

**Issue by igor-raits** [dtolnay/proc-macro-hack#33]
```
BUILDSTDERR: error[E0382]: capture of moved value: `vis`
BUILDSTDERR:    --> src/lib.rs:425:14
BUILDSTDERR:     |
BUILDSTDERR: 414 |     let crate_prefix = vis.map(|_| quote!($crate::));
BUILDSTDERR:     |                        --- value moved here
BUILDSTDERR: ...
BUILDSTDERR: 425 |         .map(|Macro { name, export_as }| {
BUILDSTDERR:     |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^ value captured here after move
BUILDSTDERR:     |
BUILDSTDERR:     = note: move occurs because `vis` has type `std::option::Option<syn::token::Pub>`, which does not implement the `Copy` trait
```

Might be related of using syn v0.15.33, but not sure. Testing this more.

**Comment by dtolnay** [comment]
Thanks! It looks like [Syn tokens have Copy if `syn/clone-impls` is enabled](https://github.com/dtolnay/syn/blob/0.15.33/src/token.rs#L236), [which we don't enable in proc-macro-hack](https://github.com/dtolnay/proc-macro-hack/blob/0.5.6/Cargo.toml#L17), but Cargo seems to enable anyway because [one of our dev-dependencies](https://github.com/dtolnay/proc-macro-hack/blob/0.5.6/Cargo.toml#L21) depends [on Syn with default features](https://github.com/dtolnay/proc-macro-hack/blob/0.5.6/demo-hack-impl/Cargo.toml#L16) which [includes clone-impls](https://github.com/dtolnay/syn/blob/0.15.33/Cargo.toml#L21).

I released proc-macro-hack 0.5.7 which depends on syn with clone-impls enabled.

**Comment by igor-raits** [comment]
Thanks! Building 0.5.7 for Fedora right away.
